src/bin/rahmen.rs

```diff
@@ -105,6 +105,15 @@ fn main() -> RahmenResult<()> {
                 .validator(|f| File::open(f))
                 .default_value("/usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf"),
         )
+        .arg(
+            Arg::new("font_size")
+                .long("font_size")
+                .takes_value(true)
+                .possible_values(&[
+                    "8", "12", "16", "20", "24", "28", "32", "40", "48", "56", "64",
```

Comment:
What about validating the value as a `f32` value instead of providing a fixed list? E.g. `.validator(|s| s.parse::<f32>())` or `f32::from_str(s))`